src/main/java/net/io/kino/controller/OrderController.java

```diff
@@ -38,12 +50,20 @@ public Order getOrder(@PathVariable Long id) {
         if (!order.isPresent()) {
             throw new ResponseStatusException(HttpStatus.NOT_FOUND);
         }
-
         return order.get();
     }
 
+    @GetMapping("/date")
+    public List<Order> getOrdersBetweenDates(@RequestParam(name = "fromDate") String from,
+                                        @RequestParam(name = "toDate") String to){
```

Comment:
Raczej nie powinien być tworzony osobny endpoint do filtrowania po dacie. Powinno to zostać w `/` i mieć dodane opcjonalne parametry "from" i "to".